CONTRIBUTING.md

```diff
@@ -7,21 +7,21 @@ You should first understand what the hell you're doing.
 
 # Setup repo
 ```
-# Public fork of Apache Spark.  Used for public pull requests.
-git remote add origin git@github.com:mesosphere/spark.git
-git fetch origin
+# Set up Mesosphere Spark, a public fork of Apache Spark, used for public pull requests.
+git clone git@github.com:mesosphere/spark.git
 
-# Apache Spark
+# Add apache spark as upstream
 git remote add upstream git@github.com:apache/spark
-git fetch upstream
+# Depending of if you want to continue work on upstream master/branch
+git pull upstream [master | <branch>]
```

Comment:
I think you can revert this back to `git fetch upstream`. This section is about adding Apache Spark as the upstream and fetching all the latest updates. Your comment about continuing work upstream would be in the `# Pull Request` section